instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+import PostHeader from './PostHeader';
+import LikesContainer from '../LikesContainer/LikesContainer';
+import CommentSection from '../CommentSection/CommentSection';
+import PropTypes from 'prop-types';
+
+const PostContainerDiv = styled.div`
+    width: 64.2rem;
+    margin: 0 auto;
+`
+const IndividualPost = styled.div`
+    border: 1px solid #E7E7E7;
+    margin: 40px 0;
+
+    img {
+        margin-top: 2px;
+    }
+`
+
+const PostContainer = props => {
+    return(
+        <PostContainerDiv>
+            {props.data.map((currentPost, index) => (
```

Comment:
If you use filter here instead of map you wouldn't need that searchData method, you could simply use the search term inside your truth check here to accomplish your search functionality.